tests/test_operation.py

```diff
@@ -125,6 +125,9 @@ def test_heisenberg(self, test_class, tol):
     def test_operation_init(self, test_class, monkeypatch):
         "Operation subclass initialization."
 
+        if test_class == qml.QubitUnitary:
```

Comment:
I had to write a separate test for the new `__init__` method (see below). I'm actually not sure why it worked before; the issue was that in line 135, the number of wires is computed and then used to construct a list of wires, but `QubitUnitary` works on any number of wires so it was yielding a list of length 0 and causing the test to fail.

Reply:
🤔